src/components/MediaSettings/MediaSettings.vue

```diff
@@ -4,249 +4,240 @@
 -->
 
 <template>
-	<NcModal v-if="modal"
-		:label-id="dialogHeaderId"
-		@close="closeModal">
+	<component :is="isDialog ? 'NcModal' : 'div'"
+		v-if="show"
+		:size="isDialog ? 'large' : undefined"
```

Comment:
Make it smaller on mobile view?
<img width="721" height="504" alt="Image" src="https://github.com/user-attachments/assets/2212f7ba-69b5-4916-88dd-4d0b315bf996" />